integration-tests/utils/terra-envs.js

```diff
@@ -1,16 +1,28 @@
 module.exports = {
   local: {
     billingProject: 'saturn-integration-test-dev',
+    dataRepoUrl: 'https://jade.datarepo-dev.broadinstitute.org',
+    snapshotColumnName: 'name',
+    snapshotId: 'f90f5d7f-c507-4e56-abfc-b965a66023fb',
+    snapshotTableName: 'tableA',
     testUrl: 'http://localhost:3000',
     workflowName: 'echo_to_file'
   },
   dev: {
     billingProject: 'saturn-integration-test-dev',
+    dataRepoUrl: 'https://jade.datarepo-dev.broadinstitute.org',
+    snapshotColumnName: 'name',
+    snapshotId: 'f90f5d7f-c507-4e56-abfc-b965a66023fb',
+    snapshotTableName: 'tableA',
     testUrl: 'https://bvdp-saturn-dev.appspot.com',
     workflowName: 'echo_to_file'
   },
   alpha: {
     billingProject: 'saturn-integration-test-alpha',
+    dataRepoUrl: 'https://data.alpha.envs-terra.bio',
+    snapshotColumnName: 'VCF_File_Name',
```

Comment:
I understand why the snapshot id would be different, but I think we should try to have the same contents of the snapshot in all environments.

Reply:
Agreed! But since QA wants us to manually test the UI until we have a UI integration test in place for these features, I think it would be best to add the tests now and then, once we have your script in place, add the test fixtures we want and change these tests.